[2023-08-30T14:25:41.150+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: schedule_news_extraction.data_process_service_container scheduled__2023-08-29T13:25:23.728317+00:00 [queued]>
[2023-08-30T14:25:41.174+0100] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: schedule_news_extraction.data_process_service_container scheduled__2023-08-29T13:25:23.728317+00:00 [queued]>
[2023-08-30T14:25:41.175+0100] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2023-08-30T14:25:41.215+0100] {taskinstance.py:1382} INFO - Executing <Task(DockerOperator): data_process_service_container> on 2023-08-29 13:25:23.728317+00:00
[2023-08-30T14:25:41.221+0100] {standard_task_runner.py:57} INFO - Started process 125267 to run task
[2023-08-30T14:25:41.233+0100] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'schedule_news_extraction', 'data_process_service_container', 'scheduled__2023-08-29T13:25:23.728317+00:00', '--job-id', '19', '--raw', '--subdir', '/home/haboubi/Desktop/final/airflow_folder/dags/news_schedule.py', '--cfg-path', '/tmp/tmpsfdoqcx1']
[2023-08-30T14:25:41.234+0100] {standard_task_runner.py:85} INFO - Job 19: Subtask data_process_service_container
[2023-08-30T14:25:41.332+0100] {task_command.py:415} INFO - Running <TaskInstance: schedule_news_extraction.data_process_service_container scheduled__2023-08-29T13:25:23.728317+00:00 [running]> on host haboubi-VivoBook-15-ASUS-Laptop-X540UB
[2023-08-30T14:25:41.515+0100] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='schedule_news_extraction' AIRFLOW_CTX_TASK_ID='data_process_service_container' AIRFLOW_CTX_EXECUTION_DATE='2023-08-29T13:25:23.728317+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-08-29T13:25:23.728317+00:00'
[2023-08-30T14:25:41.641+0100] {docker.py:333} INFO - Starting docker container from image final_dataprocess:latest
[2023-08-30T14:25:41.645+0100] {docker.py:341} WARNING - Using remote engine or docker-in-docker and mounting temporary volume from host is not supported. Falling back to `mount_tmp_dir=False` mode. You can set `mount_tmp_dir` parameter to False to disable mounting and remove the warning
[2023-08-30T14:25:45.994+0100] {docker.py:402} INFO - Welcome to the processing service
[2023-08-30T14:26:36.056+0100] {docker.py:402} INFO - There might be a problem KafkaError{code=UNKNOWN_TOPIC_OR_PART,val=3,str="Subscribed topic not available: None: Broker: Unknown topic or partition"}
[2023-08-30T14:26:39.077+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:26:39.103+0100] {docker.py:402} INFO - key in process 1
[2023-08-30T14:26:42.053+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:26:42.062+0100] {docker.py:402} INFO - key in process
[2023-08-30T14:26:42.062+0100] {docker.py:402} INFO - 
[2023-08-30T14:26:42.063+0100] {docker.py:402} INFO - 2
[2023-08-30T14:26:42.063+0100] {docker.py:402} INFO - 
[2023-08-30T14:26:43.443+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:26:43.448+0100] {docker.py:402} INFO - key in process 3
[2023-08-30T14:26:47.675+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:26:47.685+0100] {docker.py:402} INFO - key in process
[2023-08-30T14:26:47.687+0100] {docker.py:402} INFO - 
[2023-08-30T14:26:47.692+0100] {docker.py:402} INFO - 4
[2023-08-30T14:26:47.693+0100] {docker.py:402} INFO - 
[2023-08-30T14:26:54.091+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:26:54.096+0100] {docker.py:402} INFO - key in process 5
[2023-08-30T14:27:00.244+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:00.249+0100] {docker.py:402} INFO - key in process 6
[2023-08-30T14:27:05.695+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:05.699+0100] {docker.py:402} INFO - key in process 7
[2023-08-30T14:27:11.507+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:11.512+0100] {docker.py:402} INFO - key in process 8
[2023-08-30T14:27:17.682+0100] {docker.py:402} INFO - data delivered successfully
[2023-08-30T14:27:17.683+0100] {docker.py:402} INFO - topic:: auto_ingest_news
[2023-08-30T14:27:17.687+0100] {docker.py:402} INFO - key in process 9
[2023-08-30T14:27:23.384+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:23.393+0100] {docker.py:402} INFO - key in process 10
[2023-08-30T14:27:32.371+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:32.376+0100] {docker.py:402} INFO - key in process 11
[2023-08-30T14:27:40.024+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:40.029+0100] {docker.py:402} INFO - key in process 12
[2023-08-30T14:27:45.701+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:45.708+0100] {docker.py:402} INFO - key in process 13
[2023-08-30T14:27:51.289+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:51.293+0100] {docker.py:402} INFO - key in process 14
[2023-08-30T14:27:56.714+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:27:56.718+0100] {docker.py:402} INFO - key in process 15
[2023-08-30T14:28:02.851+0100] {docker.py:402} INFO - data delivered successfully
topic:: auto_ingest_news
[2023-08-30T14:28:02.855+0100] {docker.py:402} INFO - key in process 16
[2023-08-30T14:29:46.403+0100] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=schedule_news_extraction, task_id=data_process_service_container, execution_date=20230829T132523, start_date=20230830T132541, end_date=20230830T132946
[2023-08-30T14:29:46.447+0100] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-08-30T14:29:46.465+0100] {taskinstance.py:2784} INFO - 0 downstream tasks scheduled from follow-on schedule check
